web/resource/utils.go

```diff
@@ -0,0 +1,25 @@
+package resource
+
+import (
+	"fmt"
+	"net/http"
+	"sort"
+)
+
+func sortedKeys[T any](m map[string]T) []string {
```

Comment:
Please add this to core/maps.go as exported, it is not web related.

Reply:
https://github.com/darvaza-proxy/core/pull/62